python2verilog/api/context.py

```diff
@@ -56,6 +56,6 @@ def context_to_verilog_and_dump(context: ir.Context) -> tuple[str, str, str]:
     ver_code_gen, ir_root = context_to_codegen(context)
 
     module_str = ver_code_gen.get_module_str()
-    tb_str = ver_code_gen.get_testbench_str(random_ready=False)
+    tb_str = ver_code_gen.get_testbench_str(CodegenConfig())
```

Comment:
Why create a `CodegenConfig`? Aren't you trying to create a testbench?

Reply:
Good catch!

Reply:
Have fixed.